### ISSUE TITLE
Error: Column start greater than line length

### ISSUE THREAD

**Issue by denhartog** [AtomLinter/linter-pylint#165]
I upgraded pylint a week or two ago and am CONSTANTLY getting this type of error when I cut/paste/insert/delete lines in/to my files:

NOTE: (3) and (0) are different numbers corresponding to approximately where I am in my file

```
Error: Column start (3) greater than line length (0)
    at Object.rangeFromLineNumber (/Users/myUser/.atom/packages/linter-pylint/node_modules/atom-linter/lib/index.js:172:11)
    at /Users/MyUser/.atom/packages/linter-pylint/node_modules/lazy-req/index.js:20:26
    at /Users/MyUser/.atom/packages/linter-pylint/lib/main.js:132:26
    at Array.map (native)
    at /Users/MyUser/.atom/packages/linter-pylint/lib/main.js:128:14
    at process._tickCallback (node.js:392:9)
```

**Comment by summer1988** [comment]
I have the same problem with you, have you solve it?

**Comment by albireox** [comment]
I'm seeing the same problem. Any clues as to how to fix it?

**Comment by funemy** [comment]
Having same problem, is there any solution?

**Comment by glaslos** [comment]
See https://github.com/AtomLinter/linter-pylint/issues/147

**Comment by Arcanemagus** [comment]
Please try v1.2.2 and see if you are still experiencing issues with this. A check has been put in place where if the editor contents have been modified since the lint was triggered the results are discarded. This should at worst cause a slightly longer delay for messages to update as the modification discarding the results will trigger a lint of it's own.
